lldb/include/lldb/Core/Progress.h

```diff
@@ -97,12 +119,17 @@ class Progress {
   /// The title of the progress activity.
   std::string m_title;
   std::mutex m_mutex;
+  /// Set to eNonAggregateProgressReport if the progress event is aggregate;
```

Comment:
I'm a little confused by this comment.  Set to eNonAggregateProgressReport if the progress report is aggregate?

Reply:
Mistake by me 😓